cmd/sensor-config/envirosensor.go

```diff
@@ -104,3 +104,107 @@ func (n *Network) EnviroSensor(set *meta.Set, enviro string, label string) error
 
 	return nil
 }
+
+func (s Settings) EnviroSensor(set *meta.Set, name, enviro string) (Group, bool) {
+
```

Comment:
seems this is almost the same as the above `func (n *Network) EnviroSensor(set *meta.Set, enviro string, label string)`, is it possible to share code? there are other similar cases...

Reply:
The idea is we only use one output, this is aimed at keeping the existing method and adding a slightly different new method. Once the existing code is no longer needed, or the new one is rejected, then the code can be cleaned up a lot.